src/main/resources/config/application-prod.yml

```diff
@@ -19,12 +19,12 @@ spring:
         livereload:
             enabled: false
     datasource:
-        type: com.zaxxer.hikari.HikariDataSource
-        driverClassName: org.postgresql.Driver
-        url: jdbc:postgresql://localhost:5432/managementportal
-#        name: managementportal
-        username: <username>
-        password: <password>
+      type: com.zaxxer.hikari.HikariDataSource
```

Comment:
seems indentation is not consistent with rest of the file?

Reply:
You're right, I fixed it, and now its in line with the .editorconfig as well